client-core/src/key/private_key.rs

```diff
@@ -1,12 +1,12 @@
 use failure::ResultExt;
 use rand::rngs::OsRng;
 use secp256k1::SecretKey;
-use zeroize::{Zeroize, ZeroizeOnDrop};
+use zeroize::Zeroize;
 
 use crate::{ErrorKind, Result};
 
 /// Private key used in Crypto.com Chain
-#[derive(Debug, PartialEq, Zeroize, ZeroizeOnDrop)]
+#[derive(Debug, PartialEq)]
 pub struct PrivateKey(pub(super) SecretKey);
```

Comment:
We need to expose this for implementing `From<&PrivateKey> for PublicKey` in `public_key.rs`.

Reply:
Could it be implemented as `Into<PublicKey> for PrivateKey`?

Reply:
>One should only implement `Into` if a conversion to a type outside the current crate is required. Otherwise one should always prefer implementing `From` over `Into` because implementing `From` automatically provides one with a implementation of `Into` thanks to the blanket implementation in the standard library.

https://doc.rust-lang.org/nightly/std/convert/trait.Into.html

To fix this, I've implemented `From` in `private_key.rs`.